### ISSUE TITLE
Editorial updates to the document

### ISSUE THREAD

**Issue by maryjom** [w3c/wcag2ict#255]
- [x] Fix capitalization of "Success Criterion" when referring to a particular success criterion and "success criterion" elsewhere. (WCAG convention). Same with "Success Criteria" and "success criteria".
- [x] Check for Oxford commas, to ensure that they are used consistently. 
- [ ] Review all guidance to ensure consistent language is used 
- [x] Consider whether standard note in SCs that refers to Closed Functionality should link to the definition of closed functionality, the short chapter on closed functionality, or the chapter entitled "Success Criteria Problematic for Closed Functionality" (https://w3c.github.io/wcag2ict/#success-criteria-problematic-for-closed-functionality). Currently I think we point to the definition, but I wonder if it might be more helpful to point directly to the SC problematic for closed functionality section. **MJM: This is why we have renamed the "Comments on Closed Functionality" section.**  The change to that section title is now incorporated, and links go to it instead of to the definition." But there does remain whether we point to the "SC problematic..." section or the "Comments on Closed..." should be linked. Phil - think we should just keep as is for consistency, and see if we get any comments in the final public draft. 
- [x] - Review SC problematic for closed in particular and see if "Meets the intent of this SC" could be replaced with "Addresses the user needs behind this SC" or whatever language Mitch had suggested. Picked up by Phil - PR to be created to catch a few issues I spotted in SC problematic for closed section. Now in PR https://github.com/w3c/wcag2ict/pull/352 
- [x] Guidance in this document section: Remove editor's note, remove "new" when referring to the "set of software applications" and "set of documents" terms. These aren't new any more.
- **Removed** as this duplicates Issue #329 that Bruce opened. From the 14 March meeting (see [discussion on Info and Relationships](https://www.w3.org/2024/03/14-wcag2ict-minutes#t06), toward the end of that topic) regarding the section SC Problematic for Closed Functionality where Bruce indicated: "Requires" is used in 2 different ways: sometimes to restate the SC, sometimes to state why something is problematic. We need to be clearer in the language.

**Comment by maryjom** [comment]
According to the W3C Education and Outreach's style guide, "success criterion" should be lower case.  AG WG doesn't have a style guide for the WCAG standard, and their use of "Success Criterion" and "success criterion" varies throughout the standard and the additional resources, such as the Understanding docs.

**Comment by patrickhlauke** [comment]
however, in the last year or more, we've slowly trawled through WCAG to try and harmonise towards title cased "Success Criterion" whenever content gets touched/updated (though not in a consistent-enough repo-wide fashion) https://github.com/search?q=repo%3Aw3c%2Fwcag%20%22success%20criteri%22&type=code

**Comment by maryjom** [comment]
However, in the WCAG 2.2 Recommendation there are multiple instances of "success criterion" - 3 of which are in the Abstract - the very first section of the document.

**Comment by patrickhlauke** [comment]
well, you can't accuse AGWG of being internally consistent...

**Comment by pday1** [comment]
Completed editorial tasks (moved from list at start):

- [x]  "closed functionality" key term, link not right "#dfn-assistive-technology" shows up as text in the document
- [x] Edit Phil Day's affiliation to now be NCR Atleos
- [x] SC 2.1.2, Note 1, change "on the [non-web document or software]" to "in the ..."

**Comment by ChrisLoiselle** [comment]
@maryjom I attempted to search for the word "new" and "newly" within the note to remove where I saw these per the editor's task of "Guidance in this document section: Remove editor's note, remove "new" when referring to the "set of software applications" and "set of documents" terms. These aren't new any more." If you want to me to make any additional changes regarding this , please let me know.

**Comment by maryjom** [comment]
@ChrisLoiselle I've made a few additions to the removal of "new" in your PR and have merged it. Thanks for working on this item.

**Comment by maryjom** [comment]
@ChrisLoiselle I have completed a PR for the capitalization of "Success Criteria/Criterion" vs "success criteria/criterion" per the latest style guide rules which I learned from Michael Gower. Please review the PR so I can merge it.

**Comment by ChrisLoiselle** [comment]
@maryjom are we checking for whether Oxford commas are used or aren't used? I read the task that we are checking that they are used and that they must be consistent.  Thanks for confirming.

**Comment by maryjom** [comment]
@ChrisLoiselle We are checking the Oxford commas are used.

**Comment by ChrisLoiselle** [comment]
@maryjom I've gone through all markdown files and made one PR for the Oxford commas. All other files read well and were consistent per my review.

**Comment by maryjom** [comment]
@ChrisLoiselle Can you see if Phil's PR #352 addresses these things? Are there additional changes you're proposing?

**Comment by ChrisLoiselle** [comment]
@maryjom @pday1 , Phil's changes were different text than what I mentioned per the ask on this ticket, 255 , compared to PR 352. If you are happy with 352, you can ignore this research that I did.

**Comment by maryjom** [comment]
I think this can be closed, as we've agreed to publish as a TF.
